src/Dapr.Jobs/TimeSpanExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2024 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Text;
+
+namespace Dapr.Jobs;
+
+/// <summary>
+/// Provides extension methods used with <see cref="TimeSpan"/>.
+/// </summary>
+internal static class TimeSpanExtensions
+{
+    /// <summary>
+    /// Creates a duration string that matches the specification at https://pkg.go.dev/time#ParseDuration per the
+    /// Jobs API specification https://v1-14.docs.dapr.io/reference/api/jobs_api/#schedule-a-job.
```

Comment:
I find it a bit odd that the runtime didn't go with an ISO standard duration format and just used Golang's apparently arbitrary one.  I suppose it's a bit late to complain, though.

Reply:
Oh, wait, nevermind, the original spec says that the RFC and ISO durations formats should also be supported.

Reply:
Currently, I've got the SDK exposing `DateTime`s and converting to the ISO 8601 string for the backend